multiaccounts/accounts/database.go

```diff
@@ -974,3 +998,29 @@ func (db *Database) UpdateAccountToFullyOperable(keyUID string, address types.Ad
 	_, err = tx.Exec(`UPDATE keypairs_accounts SET operable = ?	WHERE address = ?`, AccountFullyOperable, address)
 	return err
 }
+
```

Comment:
How about a comment on this function's guarantees so that the user won't assume that the position is the corresponding index?